.github/workflows/windows.yml

```diff
@@ -48,9 +48,9 @@ jobs:
           # Download and install pre-built OpenBLAS library
           # Built with mingw-w64, -ucrt -static.
           # https://github.com/matthew-brett/openblas-libs/blob/ucrt-build/build_openblas.ps1
-          choco install unzip
-          pip install gdown
-          gdown --id 1G3jwxMY4ZVRqvz6owl0kmw_Innmr4ZnV
+          choco install unzip -y
+          choco install wget -y
+          wget https://github.com/jarrodmillman/scipy-ci/raw/windows-openblas/openblas_32_if.zip
```

Comment:
Perhaps best deferred to another discussion/issue, but if our OpenBLAS handling is getting a bit more tricky, does this mean we aren't going to be able to depend on `tools/openblas_support.py`.

Two reasons I ask:
- we tend to pull over changes/improvements in that file from NumPy from time to time
- we use it for our wheels infrastructure, and I currently have it involved in the draft `cibuildwheel` infrastructure

Reply:
Just a flyby comment - but I was planning to work with everyone to get our OpenBLAS / Windows builds updated, and had imagined keeping something like the existing system ...  The binary here was just a get-it-working-fast build I did when porting the Scipy build to Meson / mingw-w64.